tools/interactsh/overview.mdx

```diff
@@ -23,6 +23,22 @@ Check out [our blog introducing Interactsh](https://blog.projectdiscovery.io/int
 - Customizable Payload Length **(self-hosted)**
 - Custom SSL Certificate **(self-hosted)**
 
+## Client & Server
+The Interactsh tool comprises two main components: [`interachsh-cleint`](/tools/interactsh/running) and [`interachsh-server`](/tools/interactsh/server). Each plays a critical role in the process of detecting out-of-band vulnerabilities, but they operate in distinct manners and serve different purposes.
+
+### Interactsh Server
+* Function: Captures and records callbacks from interaction URLs.
+* Deployment: Hosted publicly to receive requests from tested systems.
+* Use Case: Ideal for those hosting their instance for privacy or control.
+
+ProjectDiscovery maintains a number of [publically accessable interactsh servers](/tools/interactsh/running#projectdiscovery-interachsh-servers) that you can use in order to only run the client for your specific use case. Alternatively, you can [self host your own interactsh server](/tools/interactsh/running#self-hosted-interactsh-server) if you want it to run on your custom domain or you need more control over the server side interactions.
```

Comment:
ProjectDiscovery maintains a number of [publicly accessable interactsh servers](/tools/interactsh/running#projectdiscovery-interachsh-servers) that you can use in order to only run the client for your specific use case. Alternatively, you can [self host your own interactsh server](/tools/interactsh/running#self-hosted-interactsh-server) if you want it to run on your custom domain or you need more control over the server side interactions.